src/main/resources/citizen/templates/document/defendantResponseReceipt.html

```diff
@@ -613,30 +613,36 @@
       <tr>
         <td colspan="2" class="keep-white-space"><strong>Bank and savings accounts:</strong></td>
       </tr>
-      {% for bankAccount in bankAccounts %}
-      <tr class="no-border">
-        <td><strong>{% if loop.length > 1 %} {{ loop.index + 1 }}. {% endif %} Type of account:</strong></td>
-        <td>
-          {{ bankAccount.type }}
-        </td>
-      </tr>
-      <tr class="no-border">
-        <td><strong>Balance:</strong></td>
-        <td>
-          {{ bankAccount.balance }}
-        </td>
-      </tr>
+      {% if bankAccounts is defined and bankAccounts is not empty %}
+        {% for bankAccount in bankAccounts | default([]) %}
```

Comment:
after the if check, do we need `| default([]` ?

Reply:
Not necessarly. Removing it.
```suggestion
        {% for bankAccount in bankAccounts %}
```